src/nuklear_panel.c

```diff
@@ -468,7 +468,10 @@ nk_panel_end(struct nk_context *ctx)
         {
             /* horizontal scrollbar */
             nk_flags state = 0;
-            scroll.x = layout->bounds.x;
+            if(layout->flags & NK_WINDOW_SCALE_LEFT)
```

Comment:
``` suggestion
            /* can't scale otherwise (horizontal scrollbar has priority) => reposition horizontal scrollbar to avoid overlap with scaler */
            if (layout->flags & NK_WINDOW_SCALE_LEFT)
```